runtime/autoload/health/nvim.vim

```diff
@@ -115,7 +115,7 @@ function! s:check_tmux() abort
 
   " check default-terminal and $TERM
   call health#report_info('$TERM: '.$TERM)
-  let cmd = 'tmux show-option -qvg default-terminal'
+  let cmd = 'tmux show-option -qvgs default-terminal'
   let out = system(cmd)
   let tmux_default_term = substitute(out, '\v(\s|\r|\n)', '', 'g')
```

Comment:
I was just about to get on this 😀  Does it set a non-zero exit code?  That'll be better than duplicating substitution.